### PR TITLE
feat: write dude commands into package.json

### PR DESCRIPTION
**Pull request by frosato-dev**

### CODE REVIEW

File: e2e/packageJson.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import path from "path";
+import { read } from "./utils";
+import { CLIENT_DIR } from "./constants";
+
+const PACKAGE_JSON_PATH = path.join(CLIENT_DIR, "package.json");
+
+
+it("should have written dude commands", () => {
+  const file = read(PACKAGE_JSON_PATH)
+  const object = JSON.parse(file)
```
Comment (fbarrailla) [inline e2e/packageJson.test.js]: maybe you can choose an explicit name here like `packageContent` or `projectConfig`

File: src/hooks/install/__tests__/writePackageJson.test.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  realpathSync: jest.fn(data => data),
+  writeFileSync: mockWriteFileSync,
+}));
+
+beforeEach(() => {
+  jest.resetAllMocks();
+  jest.resetModules();
+});
+
+it("should do nothing if no package.json found", () => {
```
Comment (fbarrailla) [inline src/hooks/install/__tests__/writePackageJson.test.js]: it should display an error ?

File: src/hooks/install/writePackageJson.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  const data = JSON.stringify(newPackageJson, null, 2);
+  const filePath = path.join(cwd, "package.json");
+
+  try {
+    fs.writeFileSync(filePath, data, { mode: 0o755 });
+    //eslint-disable-next-line
+    console.log(`✅ DudeJS successfully add commands to your package.json.`);
+  } catch (e) {
+    //eslint-disable-next-line
+    console.log(`❌ DudeJS encountered an error while writing ${filePath}`);
```
Comment (fbarrailla) [inline src/hooks/install/writePackageJson.js]: I think you should log the error too

File: e2e/packageJson.test.js (hunk 1 above)
Comment (frosato-dev) [reply e2e/packageJson.test.js]: you are definitely right, i'll change for an explicit name

**Review by fbarrailla — APPROVED** [review]

**Comment by fbarrailla** [comment]
resolve #20

**Comment by fbarrailla** [comment]
resolve #16

**Pull request merged**